packages/discord.js/src/util/Events.js

```diff
@@ -99,7 +94,7 @@ module.exports = {
   ChannelDelete: 'channelDelete',
   ChannelPinsUpdate: 'channelPinsUpdate',
   ChannelUpdate: 'channelUpdate',
-  ClientReady: 'ready',
+  ClientReady: 'clientReady',
```

Comment:
If this is going to be changed from `ready` to `clientReady`, shouldn't the types for [`Events.ClientReady`](https://github.com/discordjs/discord.js/blob/main/packages/discord.js/typings/index.d.ts#L5433) and [`ClientEvents`](https://github.com/discordjs/discord.js/blob/main/packages/discord.js/typings/index.d.ts#L5253) be updated as well?